Bio/motifs/matrix.py

```diff
@@ -227,7 +227,7 @@ def gc_content(self):
 
     def reverse_complement(self):
         values = {}
-        if "U" in self.alphabet.letters:
+        if isinstance(self.alphabet, Alphabet.RNAAlphabet):
```

Comment:
We *probably* don't have to worry about this in ``Bio.motifs``, but this could use this private function in ``Bio.Alphabet`` to deal with the messy stuff about gaps etc:

```
if isinstance(Alphabet._get_base_alphabet(self.alphabet), Alphabet.RNAAlphabet):
```